app/controllers/mentor_controller.rb

```diff
@@ -0,0 +1,23 @@
+class MentorController < ApplicationController
+
+
+  def new
+      @mentor=Mentor.new
+  end
+
+  def create
+    @mentor=Mentor.new
+    @user = User.find(session[:user_id])
+    @mentor.user=@user
+    @mentor.email=@user.email
```

Comment:
this is unnecessary.  If you need the email of a mentor elsewhere, you can do mentor.user.email